cclib/bridge/cclib2pyscf.py

```diff
@@ -301,6 +325,26 @@ def cclibfrommethods(
         pyscf.data.elements.COMMON_ISOTOPE_MASSES[atom_no] for atom_no in attributes["atomnos"]
     ]
 
+    attributes["atommasses"] = []
+    for atom_index in range(attributes["natom"]):
+        # If an isotope mass has been given for this atom, we can use that.
+        if (
+            mol.elements[atom_index] in mol.nucprop
```

Comment:
Can you factor out `mol.elements[atom_index]` into a variable? outside the conditional?  Only because I see it used a bunch.